docs/document/docs/guides/table-readonly-table-customization.md

```diff
@@ -0,0 +1,514 @@
+      field: 'address',
+      render: renderAddress,
+    },
+];
+
+const data = [
+    {
+        name: 'John Brown',
+        gender: 'male',
+        address: 'osaka-japan',
+    },
+    {
+        name: 'Jim Green',
+        gender: 'female',
+        address: 'tokyo-japan',
+    },
+    {
+        name: 'Joe Black',
+        gender: 'male',
+        address: 'hochiminh-vietnam',
+    },
+];
+
+const table = new Kuc.Table({columns, data});
+const space = kintone.app.record.getSpaceElement('space');
+space.appendChild(table);
+  ```
+</details>
+
+### Multiple components in one cell
+![multi-components](assets/two-component-in-cell.png)
+
+You can put multiple component in one table cell using `columns.render` function.</br>
+The example code below will display two dropdowns (city and country) in the same cell.
+<details>
+  <summary>Show code</summary>
+
+  ```js
+  const renderAddress = (cellData, rowData) => {
+    // the format of cellData: "city-country";
+    const city = cellData.split('-')[0];
+    const country = cellData.split('-')[1];
+
+    const dropdownCity = new Kuc.Dropdown({
+      items: [
+        {
+          label: 'Tokyo',
+          value: 'tokyo',
+        },
+        {
+          label: 'Osaka',
+          value: 'osaka',
+        },
+        {
+          label: 'Ho Chi Minh',
+          value: 'hochiminh',
+        },
+      ],
+      value: city,
+    });
+    dropdownCity.addEventListener('change', (event) => {
+      const _country = rowData.address.split('-')[1];
+      event.detail.value = `${event.detail.value}-${_country}`;
+    });
+
+    const dropdownCountry = new Kuc.Dropdown({
+      items: [
+        {
+          label: 'Viet Nam',
+          value: 'vietnam',
+        },
+        {
+          label: 'Japan',
+          value: 'japan',
+        },
+      ],
+      value: country,
+    });
+    dropdownCountry.addEventListener('change', (event) => {
+      const _city = rowData.address.split('-')[0];
+      event.detail.value = `${_city}-${event.detail.value}`;
+    });
+
+    const container = document.createElement('div');
+    container.style.display = 'flex';
+    container.appendChild(dropdownCountry);
+    container.appendChild(dropdownCity);
+
+    return container;
+  };
+
+  const data = [
+    {
+      name: 'John Brown',
+      gender: 'male',
+      address: 'osaka-japan',
+    },
+    {
+      name: 'Jim Green',
+      gender: 'female',
+      address: 'tokyo-japan',
+    },
+    {
+      name: 'Joe Black',
+      gender: 'male',
+      address: 'hochiminh-vietnam',
+    },
+  ];
+
+  const columns = [
+    {
+      title: 'Name',
+      field: 'name',
+    },
+    {
+      title: 'Address',
+      field: 'address',
+      render: renderAddress,
+    },
+  ];
+
+const table = new Kuc.Table({columns, data});
+const space = kintone.app.record.getSpaceElement('space');
+space.appendChild(table);
+  ```
+</details>
+
+
+### Dependent columns
+![dependent-columns](assets/dependent-columns.gif)
```

Comment:
Please change the image with the English version name of Viet Nam and Ho Chi Minh.
```suggestion
![dependent-columns](assets/dependent-columns.gif)

```

Reply:
I changed the image